lib/components/project/data/project.dart

```diff
@@ -1,30 +0,0 @@
-import 'package:cloud_firestore/cloud_firestore.dart';
```

Comment:
Not sure why but git thinks I deleted this file but was only changed from one folder to another